app/src/main/kotlin/io/intrepid/skotlinton/base/BaseFragment.kt

```diff
@@ -8,16 +8,15 @@ import android.support.v4.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-
 import butterknife.ButterKnife
 import butterknife.Unbinder
 import io.intrepid.skotlinton.SkotlintonApplication
 import timber.log.Timber
 
-abstract class BaseFragment<P : BaseContract.Presenter> : Fragment(), BaseContract.View {
+abstract class BaseFragment<out P : BaseContract.Presenter> : Fragment(), BaseContract.View {
 
     protected val skotlintonApplication: SkotlintonApplication
-        get() = activity?.application as SkotlintonApplication
+        get() = activity!!.application as SkotlintonApplication
```

Comment:
@f2rf2r mentioned on my last PR that it might be nice to have some sort of `val a = activity!!` to wrap this, but I couldn't come up with a good name for it. Mostly made this change to flag for discussion, but we might also decide that this is the style we want to use in fragments moving forward so it might be worth leaving in as a demo.

Not also that the same thing applies for `context`.